crates/libr/src/sys/windows/types.rs

```diff
@@ -208,29 +208,20 @@ impl structRstart {
     }
     #[inline]
     pub fn RstartVersion(&self) -> ::std::os::raw::c_int {
-        unsafe { ::std::mem::transmute(self._bitfield_1.get(16usize, 16u8) as u32) }
+        self._bitfield_1.get(16usize, 16u8) as ::std::os::raw::c_int
     }
     #[inline]
     pub fn set_RstartVersion(&mut self, val: ::std::os::raw::c_int) {
-        unsafe {
-            let val: u32 = ::std::mem::transmute(val);
-            self._bitfield_1.set(16usize, 16u8, val as u64)
-        }
+        self._bitfield_1.set(16usize, 16u8, val as u64)
```

Comment:
```suggestion
        self._bitfield_1.set(16usize, 16u8, i32::cast_unsigned(val) as u64)
```

Again, reading the help message closely.